src/components/Popup/Popup.js

```diff
@@ -93,14 +93,20 @@ function Popup(props) {
                 let rawImg = value.data.image;
                 var name = value.data.name;
                 var desc = value.data.description;
+                const correct_num = Number(name.charAt(name.length - 1)) + 1;
+                const updated_name = name.replace(
+                  name.charAt(name.length - 1),
```

Comment:
Add comment here defining the problem with numbering and that you are solving it here

Reply:
updated